### ISSUE TITLE
MigtrationManager.listApplied fails without ensureStructure() called before

### ISSUE THREAD

**Issue by theanimal666** [orientechnologies/orientjs#245]
Hi,

Following code:
```javascript
const manager = new OrientDB.Migration.Manager({
	db,
	dir: this.config.migrations.dir,
});

// uncommentig this makes it work
// await manager.ensureStructure();

const appliedMigs = await manager.listApplied();
this.log.info('Already applied migrations:', appliedMigs);
```
fails with:
```
 OrientDB.RequestError: No such class: Migration
      at ODatabase.<anonymous> (node_modules/orientjs/lib/db/class/index.js:297:69)
      at ODatabase.tryCatcher (node_modules/orientjs/node_modules/bluebird/js/main/util.js:26:23)
      at Promise._settlePromiseFromHandler (node_modules/orientjs/node_modules/bluebird/js/main/promise.js:503:31)
      at Promise._settlePromiseAt (node_modules/orientjs/node_modules/bluebird/js/main/promise.js:577:18)
      at Async._drainQueue (node_modules/orientjs/node_modules/bluebird/js/main/async.js:128:12)
      at Async._drainQueues (node_modules/orientjs/node_modules/bluebird/js/main/async.js:133:10)
      at Immediate.Async.drainQueues (node_modules/orientjs/node_modules/bluebird/js/main/async.js:15:14)
```

Possible solution in lib/migration/manager.js
```javascript
MigrationManager.prototype.listApplied = function () {
  // the this.ensureStructure() call is missing here
  return this.enshureStructure()
   .bind(this)
   .then(function () {
       return this.db.class.get(this.className)
      .bind(this);
   })
  .then(function (migrations) {
    return migrations.list(Infinity);
  });
};
```

sorry but i do not have time to clone, patch, test and create PR but the fix should be a no-brainer on your side.

kind regards,
Chirs

**Comment by maggiolo00** [comment]
Hi @theanimal666 

Thanks for the Feedback. I've fixed it and pushed in master

Thanks

**Comment by maggiolo00** [comment]
@theanimal666 

just released a new version 2.2.5 with your fix
